.github/workflows/ci.yml

```diff
@@ -103,8 +105,13 @@ jobs:
       - name: Change admin password to make it easier for Cypress tests
         run: ddev drush upwd "$CYPRESS_ADMIN_USERNAME" "$CYPRESS_ADMIN_PASSWORD"
 
-      - name: Cypress Test
-        run: ddev cypress-run
+      # - name: Cypress Test
+      #   run: ddev cypress-run
```

Comment:
Please remove the comments.

Reply:
I kept it for review. so if we just want to run cypress without percy, we can use this. I will remove it from here and add it to the documentation.